bitnami/mongodb/values.yaml

```diff
@@ -354,14 +430,35 @@ sidecars: {}
 ## extraVolumes:
 ##   - name: extras
 ##     emptyDir: {}
+#
+#
+#
 extraVolumeMounts: []
 extraVolumes: []
+## Uncomment to create the volumes and volume mount for the TLS certs
+## extraVolumeMounts:
+#  - name: certs
+#    mountPath: /certs
+#
+## extraVolumes:
+#  - name: certs
+#    emptyDir: {}
+#  - name: certs-volume
+#    secret:
+#      secretName: ca-clients
+#      items:
+#      - key: mongodb-ca-cert
+#        path: mongodb-ca-cert
+#        mode: 511
+#      - key: mongodb-ca-key
+#        path: mongodb-ca-key
+#        mode: 511
 
 ## MongoDB Pod Disruption Budget configuration
 ## ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb/
 ##
 pdb:
-  create: false
+  create: true
```

Comment:
Changing the default values will requerie a major version, please do noo thcnage them

Reply:
done

Reply:
This is not done